defaults/main.yml

```diff
@@ -1,2 +1,4 @@
 ---
 supervisord_config_dir: "/etc/supervisor.d"
+version: 3.3.1
```

Comment:
The idea of variables in roles is so you can override them in inventory. `version` is not specific enough.